academy/4-my-own-chain/reject-game.md

```diff
@@ -157,7 +157,167 @@ You can confirm that your project at least compiles [with](https://docs.starport
 $ starport chain build
 ```
 
+## Interact with the CLI
 
+Time to see if it is possible to reject a game. First, is it possible to reject the current game from the command-line?
+
+```sh
+$ checkersd tx checkers --help
+```
+
+Which prints:
+
+```
+...
+Available Commands:
+...
+  reject-game Broadcast message rejectGame
+```
+
+`reject-game` is the command. What is its syntax?
+
+```sh
+$ checkersd tx checkers reject-game --help
+```
+
+Which prints:
+
+```
+...
+Usage:
+  checkersd tx checkers reject-game [idValue] [flags]
+```
+
+Simple. Let's have Alice, who played poorly in this game `0`, try to reject it:
+
+```sh
+$ checkersd tx checkers reject-game 0 --from $alice
+```
+
+Which returns:
+
+```
+...
+raw_log: '[{"events":[{"type":"message","attributes":[{"key":"action","value":"RejectGame"},{"key":"module","value":"checkers"},{"key":"action","value":"GameRejected"},{"key":"Creator","value":"cosmos1gml05nvlhr0k27unas8mj827z6m77lhfpzzr3l"},{"key":"IdValue","value":"0"}]}]}]'
+```
+
+Oh.
```

Comment:
Just "Oh"? :-) Maybe add a little elaboration to the Oh.

Reply:
```suggestion
Oh. It went through, against our expectations.
```
I wanted a flagrant understatement before the explanation in the box.